templates/common/common_head.html

```diff
@@ -24,3 +24,6 @@
 <link rel="stylesheet" href="{% static 'css/patterns.css' %}">
 
 <link href="{% static 'vendor/aos-2.3.4-dist/css/aos.css' %}" rel="stylesheet">
+<link rel="stylesheet" href="https://cdn.datatables.net/1.13.7/css/dataTables.bootstrap4.min.css">
+<script src="https://cdn.datatables.net/1.13.7/js/jquery.dataTables.min.js"></script>
+<script src="https://cdn.datatables.net/1.13.7/js/dataTables.bootstrap4.min.js"></script>
```

Comment:
Do you think that it should go in the common header and not extra header in the specific html file that it is used?

Reply:
I am not sure about this. It makes sense to have all the css and js files in one place as that is more manageable. And I followed this because this is how it is being done right now in the project. There is also merit in the thought that if some files are only used in certain pages, then maybe it is less costly in terms of fetching resources to include them in only that specific place.